get_aqi.py

```diff
@@ -1,32 +1,55 @@
 #!/usr/bin/python3
 
+import os
 import argparse
 import datetime
 import time
+import logging
 from collections import deque
+from threading import Thread
 
 import paho.mqtt.publish
+from prometheus_client import start_http_server, Gauge, Histogram
+
 from aqipy import aqi_cn, aqi_us, caqi_eu
 from sds011 import SDS011
 
+logging.basicConfig(
+    format='%(asctime)s.%(msecs)03d %(levelname)-8s %(message)s',
+    level=logging.INFO,
+    handlers=[logging.FileHandler("sds011_exporter.log"),
+              logging.StreamHandler()],
+    datefmt='%Y-%m-%d %H:%M:%S')
+
+DEBUG = os.getenv('DEBUG', 'false') == 'true'
 
 def parse_args():
     parser = argparse.ArgumentParser(description="Measure air quality using an SDS011 sensor.")
 
-    parser.add_argument("--country", "-c", choices=["CN", "EU", "US"], default="EU", metavar="COUNTRY", help="country code (ISO 3166-1 alpha-2) used to compute AQI. Currently accepted values (default: EU) : 'CN' (AQI Mainland China), 'EU' (CAQI) and 'US' (AQI US)")
-    parser.add_argument("--delay", "-d", default=1200, metavar="SECONDS", type=int, help="seconds to pause after getting data with the sensor before taking another measure (default: 1200, ie. 20 minutes)")
+    parser.add_argument("-b", "--bind", metavar='ADDRESS', default='0.0.0.0', help="Specify alternate bind address [default: 0.0.0.0]")
+    parser.add_argument("-p", "--port", metavar='PORT', default=8000, type=int, help="Specify alternate port [default: 8000]")
+    parser.add_argument("--country", "-c", choices=["CN", "EU", "US"], default="US", metavar="COUNTRY", help="country code (ISO 3166-1 alpha-2) used to compute AQI. Currently accepted values (default: EU) : 'CN' (AQI Mainland China), 'EU' (CAQI) and 'US' (AQI US)")
+    parser.add_argument("--delay", "-d", default=10, metavar="SECONDS", type=int, help="seconds to pause after getting data with the sensor before taking another measure (default: 1200, ie. 20 minutes)")
     parser.add_argument("--log", "-l", metavar="FILE", help="path to the CSV file where data will be appended")
     parser.add_argument("--measures", "-m", default=3, metavar="N", type=int, help="get PM2.5 and PM10 values by taking N consecutive measures (default: 3)")
     parser.add_argument("--mqtt-hostname", "-n", metavar="IP/HOSTNAME", help="IP address or hostname of the MQTT broker")
     parser.add_argument("--mqtt-port", "-r", default="1883", metavar="PORT", type=int, help="Port number of the MQTT broker (default: '1883')")
-    parser.add_argument("--mqtt-base-topic", "-i", default="aqi", metavar="TOPIC", help="Parent MQTT topic to use (default: 'aqi')")
+    parser.add_argument("--mqtt-base-topic", "-i", default="sds011", metavar="TOPIC", help="Parent MQTT topic to use (default: 'aqi')")
```

Comment:
I prefer to keep the default MQTT topic to `aqi` as it reflects better the name of the data than `sds011` which is the name of the sensor used

```suggestion
    parser.add_argument("--mqtt-base-topic", "-i", default="aqi", metavar="TOPIC", help="Parent MQTT topic to use (default: 'aqi')")
```